cli-typescript/src/utils/auth/okta.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * Authenticate using okta
+ *
+ * TODO: we forked this file from cli/ to unblock eth tooling. Refactor this into a shared lib.
+ */
+
+import axios from 'axios';
+import { createHash, randomBytes } from 'crypto';
+import express from 'express';
+import open from 'open';
+import { stringify } from 'querystring';
+
+export const OKTA = {
+  oktaOrgUrl: 'https://magiceden.okta.com',
+  clientId: '0oaczwrqrdhJTKiuC696',
```

Comment:
Is this OK to be public?

Reply:
Yes. [OKTA ClientIds are not secrets](https://devforum.okta.com/t/is-authorization-server-id-or-client-id-supposed-to-be-secret/24455)